srtcore/core.cpp

```diff
@@ -9512,7 +9510,7 @@ int CUDT::processData(CUnit* in_unit)
                 }
                 else
                 {
-                    LOGC(mglog.Error, log << CONID() << "No room to store incoming packet: offset="
+                    LOGC(mglog.Warn, log << CONID() << "No room to store incoming packet: offset="
```

Comment:
This one I wanted to use to dive deeper into #1246.
Can we leave it at the "Error" level fow now?

Reply:
Leaving it as Warn. Will see how it goest with 1246.